tools/docker/docker.go

```diff
@@ -27,13 +27,18 @@ import (
 
 // RunOptions defines optional arguments for RunServer.
 type RunOptions struct {
-	ID     chan<- string
-	Output io.Writer
+	ID         chan<- string
+	Output     io.Writer
+	DockerArgs []string
```

Comment:
instead, I think adding a Mount member here is better. Keeping `Args` the way it is would be fine.